alacritty_terminal/src/term/mod.rs

```diff
@@ -165,8 +165,8 @@ impl<'a, C> Iterator for RenderableCellsIter<'a, C> {
                 let cell = self.inner.next()?;
                 let cell = RenderableCell::new(self, cell);
```

Comment:
Since this doesn't affect our cursor path, does this mean we still need all our special selection logic solely to make the block cursor work?

Reply:
I'm not quite sure about what are you talking about here, unfortunately. What do you mean by 'work', like expand over normal chars? If that's the case, I think we may apply all coloring where block cursor is to the cell below it? And then in selection just not mark this cell as selected?

Reply:
What is the reason why the logic for checking if a wide char spacer is selected is still in this code? It shouldn't be necessary anymore.

Reply:
Well, you can't really just remove it, it'll require a slight rewrite, I'll look into it once I have time.